hassio-google-drive-backup/backup/static/css/static.css

```diff
@@ -1,79 +1,235 @@
 body {
-    display: flex;
-    min-height: 100vh;
-    flex-direction: column;
-  }
-  
-  main {
-    flex: 1 0 auto;
-  }
-  
-  .icon-block .material-icons {
-      font-size: inherit;
-  }
-  
-  .indented-li {
-    padding-left: 15px;
-    padding-top: 5px;
-    padding-bottom: 5px;
-  }
-  
-  .important-inline {
-    display: inline !important;
-  }
-  
-  .detail-node{ 
-    margin-top: 0px;
-    margin-bottom: 5px;
-  }
-  
-  .top-btn {
-    box-shadow: none;
-    -webkit-box-shadow: none;
-    margin-right: 10px;
-  }
-
-  .danger-button {
-    color: white;
-    background-color: rgb(153, 0, 0);
-  }
-
-  .error_card a {
-    color: white;
-    text-decoration: underline;
-  }
-  
-  .default-hidden {
-    display: none;
-  }
-  
-  .console-error {
-    color: #b30000
-  }
-  
-  .console-warning {
-    color: #a55d00
-  }
-  
-  .console-default {
-    color: #ffffff
-  }
-  
-  .console-debug {
-    color: #3141d1
-  }
-  
-  .console-critical {
-    color: #810000
-  }
-
-  .colpick {
-    z-index: 9999;
-  }
-
-  #toast-container {
-    top: auto !important;
-    left: auto !important;
-    bottom: 10%;
-    right: 7%;
-  }
\ No newline at end of file
+  display: flex;
```

Comment:
> * I really like the new snapshot card and its a big improvement over the old one.  I'm less confidant about the mechanics of a 3 column layout.  Many users have long custom snapshot names where the date or otherwise relevant part of the name would get truncated from the current size.  I was thinking to get around this we could:
>   
>   * (probably fine) Use a 2-column layout instead for large format screens.  This still has the downside that snapshots aren't in an "obvious" date-order, but I think there would be enough room for most user's custom names.
>   * (Better) Use a single column layout, but switch to 2 column if there are more snapshots than would fit in the vertical page space without scrolling.  I'm not sure if there is an elegant way to make materialize's column rules do this.

@sabeechen I've added a small tooltip if you hover your mouse over the truncated text.

As for the 3 column layout, I think that we could leave it like it is. I've made the size of each card similar to what HA is doing:

![image](https://user-images.githubusercontent.com/9013072/110257081-36f1a880-7f6a-11eb-9eeb-01f1d838bfec.png)

vs

![image](https://user-images.githubusercontent.com/9013072/110257107-52f54a00-7f6a-11eb-8ac1-f3ebe5f96767.png)

Also note that I'm using `col s12 m12 l6 xl4`, which means that we only get 3 columns on xl screens.

> Many users have long custom snapshot names where the date or otherwise relevant 

That is why I've added a text which says when the screenshot was taken 😄 : 
![image](https://user-images.githubusercontent.com/9013072/110257193-a49dd480-7f6a-11eb-86cb-5083d55fba62.png)

---

Here is a screenshot if I switch to 2 columns on xl screens:

![image](https://user-images.githubusercontent.com/9013072/110257208-c0a17600-7f6a-11eb-9339-66e5f24a70c3.png)

I have a preference for the 3 columns layout, but 2 also looks clean.

So if you still prefer 2, I will switch no problem. Let me know what you think!